index.js

```diff
@@ -111,12 +108,11 @@ function Launder(options) {
     }
     return s;
 
-
     function fixUrl(href) {
-      if (href.match(/^(((https?|ftp)\:\/\/)|mailto\:|\#|([^\/\.]+)?\/|[^\/\.]+$)/)) {
+      if (href.match(/^(((https?|ftp|mailto):\/\/)|#|([^/.]+)?\/|[^/.]+$)/)) {
```

Comment:
mailto:// is not the syntax, it is just mailto:foo@bar.com, so the meaning has been altered.

Reply:
doh.